releases/sui/2025-01-ITS-v1.1.0.md

```diff
@@ -0,0 +1,158 @@
+# EVM ITS v1.1.0
+
+|  | **Owner** |
+|-----------|------------|
+| **Created By** | @milapsheth <milap@interoplabs.io>, @Foivos <foivos@interoplabs.io> |
+| **Deployment** | @blockchainguyy <ayush@interoplabs.io> |
+
+| **Network** | **Deployment Status** | **Date** |
+|-------------|----------------------|----------|
+| **Devnet Amplifier** | - | TBD |
+| **Stagenet** | - | TBD |
+| **Testnet** | - | TBD |
+| **Mainnet** | - | TBD |
+
+[Release](https://github.com/axelarnetwork/axelar-cgp-sui/releases/tag/v1.1.0)
+
+[Changelog](https://github.com/axelarnetwork/interchain-token-service/blob/v/CHANGELOG.md#210)
+
+## Deployment
+
+- Ensure that ITS Hub has been upgraded to [v1.1.0](../cosmwasm/2025-01-ITS-v1.1.0.md) first.
+- The rollout involves deploying Sui ITS contracts version `v1.1.0` on Sui.
+
+```bash
+# Clone latest main and update deps
+npm ci
+```
+
+Create an `.env` config. Use `all` for `CHAINS` to run the cmd for every EVM chain, or set a specific chain.
+
+```yaml
+MNEMONIC=<wasm-deployer-key>
+PRIVATE_KEY=<sui-deployer-key>
+PRIVATE_KEY_TYPE="mnemonic" # Optional
+SIGNATURE_SCHEME=secp256k1
+ENV=<devnet-amplifier|stagenet|testnet|mainnet>
+CHAIN=sui
+```
+
+### Deploy move contracts
+- Ensure that there isn’t a move directory in Sui directory, if it exists delete it.
+```bash
+rm -rf ./sui/move
+```
+
+- Deploy the ITS Contract and required depndencies on Sui (remember to save `sui/move` files for ITS deployment)
+
+```bash
+node sui/deploy-contract.js deploy ABI
+node sui/deploy-contract.js deploy ITS
```

Comment:
The package name is `InterchainTokenService`, not `ITS`, probably rebase on https://github.com/axelarnetwork/axelar-contract-deployments/pull/517.